File: MOST_MMC_BaseBizComponent/src/com/tsb/most/basebiz/parm/codes/SearchBrandModelCodeParm.java
package com.tsb.most.basebiz.parm.codes;

import com.tsb.most.framework.bizparm.BaseBizParm;

public class SearchBrandModelCodeParm extends BaseBizParm {
	private String brandCd;
	private String brandNm;
	private String modelNm;
	private String modelCd;
	
	public String getBrandNm() {
		return brandNm;
	}
	public void setBrandNm(String brandNm) {
		this.brandNm = brandNm;
	}
	public String getModelNm() {
		return modelNm;
	}
	public void setModelNm(String modelNm) {
		this.modelNm = modelNm;
	}
	public String getModelCd() {
		return modelCd;
	}
	public void setModelCd(String modelCd) {
		this.modelCd = modelCd;
	}
	public String getBrandCd() {
		return brandCd;
	}
	public void setBrandCd(String brandCd) {
		this.brandCd = brandCd;
	}

}

File: MOST_MMC_Client/MOST_SPT/app/model/codes/SearchBrandModelCodeParm.js
Ext.define('MOST.model.codes.SearchBrandModelCodeParm',{
	extend: 'MOST.model.foundation.parm.BizParm',
	fields:[
		{
		    name:'brandCd',
            type:'string'
        },{
			name:'brandNm',
			type:'string'
		},{
            name:'modelCd',
            type:'string'
        },{
			name:'modelNm',
			type:'string'
		}
	]
});